### PR TITLE
Add commas in UI to long numbers, local date format

### PR DESCRIPTION
**Pull request by shellderp**
image

### CODE REVIEW

File: service/web/tabs/app/src/containers/BackfillStatusContainer.tsx [hunk 1]
```diff
@@ -435,8 +450,25 @@ class BackfillStatusContainer extends React.Component<
...
-                    {event_log.occurred_at}
+                  <td
+                    style={{
+                      paddingTop: 0,
+                      paddingBottom: 0,
+                      whiteSpace: "nowrap",
+                    }}
+                  >
+                    {new Date(event_log.occurred_at)
+                      .toLocaleString("en-CA", {
```
Comment (mpawliszyn) [inline service/web/tabs/app/src/containers/BackfillStatusContainer.tsx:461]: Do we want to create a few of these?

**Review by mpawliszyn — APPROVED** [review]
As long as it renders, great. Is en_CA the default then? We do have people all of the world using this LOL

**Comment by shellderp** [comment]
> As long as it renders, great. Is en_CA the default then? We do have people all of the world using this LOL

lol EN_CA just has a nice format. Trying to get JS to produce dates is hard

File: service/web/tabs/app/src/containers/BackfillStatusContainer.tsx (hunk 1 above)
Comment (shellderp) [reply service/web/tabs/app/src/containers/BackfillStatusContainer.tsx:461]: meh

**Pull request merged**